lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,8 +161,131 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
+    #
+    def create_shipment(origin, destination, packages, options = {})
+      options = @options.update(options)
+      packages = Array(packages)
+
+      begin
+        request = build_shipment_request(origin, destination, packages, options)
+        logger.debug(request) if logger
+
+        logger.debug(confirm_response) if logger
+
+        response = commit(save_request(request), (options[:test] || false))
+        parse_ship_response(response)
+      rescue RuntimeError => e
+        raise "Could not obtain shipping label. #{e.message}."
```

Comment:
Yah we probably should remove it from both places. Although if we are lucky pinging @jmorton (guy who wrote it according to git blame) will lend some insight.